v3io/dataplane/request.py

```diff
@@ -117,13 +117,19 @@ def encode_get_object(container_name, access_key, kwargs):
 
 
 def encode_put_object(container_name, access_key, kwargs):
-    headers = None
+    headers = {
+        "Content-Type": "application-octet-stream",
```

Comment:
This isn't really a part of the fix. I just noticed it was expected according to [the docs](https://www.iguazio.com/docs/latest-release/services/data-layer/reference/web-apis/simple-object-web-api/data-object-operations/#appending-data-examples).